charts/rustrial-aws-eks-iam-auth-controller/templates/additional-iamidentity-mapping.yaml

```diff
@@ -0,0 +1,13 @@
+{{- range .Values.additionalIdentities }}
+apiVersion: iamauthenticator.k8s.aws/v1alpha1
+kind: IAMIdentityMapping
+metadata:
+  name: {{ .name }}
+  annotations:
+    "helm.sh/hook": post-install
```

Comment:
Can you please elaborate why it is necessary to create those resources in a "post-install" hook?

Reply:
We want the CRD to present first to prevent failures

Reply:
AFAIK, CRDs (in `/crds/`) are always first installed by Helm before applying any other resources, no need to use hooks for that purpose.

Reply:
OK great I'm removing this